Vagrantfile

```diff
@@ -160,6 +180,20 @@ Vagrant.configure(2) do |config|
     mount_folders(override, [])
   end
 
+  config.vm.provider "libvirt" do |libvirt, override|
+      # Default box with libvirt support
```

Comment:
Same for this comment, it's just the "what" of this block, and it's simple enough already to not warrant it.